File: scala/Penis.scala
object Penis {
    def main(penis: Array[String]) {
        if (penis.length <= 0) { // Not long enough
            println("GIVE ME A FILE TO FUCK UP");
            return;
        }

        val file = new java.io.File(penis(0));

        if (!file.exists) {
            println("GIVE ME A REAL FILE TO FUCK UP");
            return;
        }

        val text = scala.io.Source.fromFile(file).mkString;

        val writer = new java.io.PrintWriter(file);
        writer.write(text.replaceAll("\\S+", "penis"));
        writer.close();
    }
}

File: clojure/penis.clj
(let [filename (first *command-line-args*)]
    (if (nil? filename)
        (println "feed me a file")
        (let [file (clojure.java.io/as-file filename)]
            (if (not (.exists file))
                (println "feed me a real file")
                (let [text (slurp file)]
                    (spit file
                        (clojure.string/replace text #"\S+" "penis")
                    )
                )
            )
        )
    )
)
